### PR TITLE
BGDIINF_SB-1371: Added datetime query in items endpoint

### PR DESCRIPTION
**Pull request by ltshb**
Fixing up PR #68

### CODE REVIEW

**Comment by rebert** [comment]
make migrate did not work ...,
`django.db.migrations.exceptions.NodeNotFoundError: Migration stac_api.0020_auto_20201109_0749 dependencies reference nonexistent parent node ('stac_api', '0019_auto_20201106_1351')
`

**Comment by ltshb** [comment]
> make migrate did not work ...,
> `django.db.migrations.exceptions.NodeNotFoundError: Migration stac_api.0020_auto_20201109_0749 dependencies reference nonexistent parent node ('stac_api', '0019_auto_20201106_1351') `

Are you sure that you are up to date and have a clean db state ? there is no 0020_auto_20201109_0749 dependencies on this PR and the latest migration file is 0019 ? are you sure that you have a clean repo ?

File: app/stac_api/models.py [hunk 1]
```diff
@@ -221,37 +219,23 @@ def update_temporal_extent(self, item_properties_datetime):
...
-                self.cache_start_datetime = item_properties_datetime
-                self.save()
-            elif self.cache_end_datetime is None:
-                self.cache_end_datetime = item_properties_datetime
-                self.save()
-            elif item_properties_datetime > self.cache_end_datetime:
-                self.cache_end_datetime = item_properties_datetime
-                self.save()
-
-            # TODO: Delete-case is yet to be implemented!
```
[the remark this answers is not in the document]
Comment (hansmannj) [reply app/stac_api/models.py:244]: OK, I see, the comment is still in place in the save function, so, forget about the above comment ;-)

**Review by hansmannj — APPROVED** [review]
Nice (again) ;-)
Thanks!

**Pull request merged**